src/pages/permission-management/user/list/Index.js

```diff
@@ -0,0 +1,587 @@
+import React, { PureComponent, Fragment } from 'react';
+import router from 'umi/router';
+import { connect } from 'dva';
+import {
+  List,
+  Card,
+  Row,
+  Col,
+  Radio,
+  Input,
+  InputNumber,
+  Progress,
+  Button,
+  Icon,
+  Dropdown,
+  Menu,
+  Avatar,
```

Comment:
Codacy found an issue: ['Avatar' is defined but never used.](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=6006324)